packages/discord.js/typings/index.d.ts

```diff
@@ -3869,6 +3878,7 @@ export interface CommandInteractionOption<Cached extends CacheType = CacheType>
   member?: CacheTypeReducer<Cached, GuildMember, APIInteractionDataResolvedGuildMember>;
   channel?: CacheTypeReducer<Cached, GuildBasedChannel, APIInteractionDataResolvedChannel>;
   role?: CacheTypeReducer<Cached, Role, APIRole>;
+  attachment?: CacheTypeReducer<Cached, MessageAttachment>;
```

Comment:
Not sure there's any need for this line to use `CacheTypeReducer` with only one possible type

Reply:
fixed